File: Aula_1/Pessoa.cpp
#include <iostream>
#include <string>

using namespace std;
class Pessoa
{
public:
    string nome;
    int idade;
    void dizer_ola()
    {
        cout << "Olá, meu nome é "
            << nome << " e tenho "
            << idade << " anos!"<< endl;
    }
};
int main()
{
    Pessoa p;
    p.nome = "João";
    p.idade = 35;
    p.dizer_ola();
    return 0;
}

File: Aula_1/Pessoa.java
package lpoo2;


public class Pessoa {

    public String nome;
    public int idade;

    public void dizerOla() {
        System.out.println("Olá,  meu nome é " 
                + nome + " e tenho " 
                + idade + " anos");
    }

    public static void main(String[] args) {
        Pessoa p = new Pessoa();
        p.nome = "João";
        p.idade = 25;
        p.dizerOla();
    }
}

